lms/djangoapps/course_goals/toggles.py

```diff
@@ -0,0 +1,21 @@
+"""
+Toggles for course goals
```

Comment:
Feels like `ENABLE_COURSE_GOALS` should maybe live in this file too, but that's not your doing and no need to change that here.

Is part of this feature work going to eventually be removing `ENABLE_COURSE_GOALS` and turning this all on by default?